README.md

```diff
@@ -20,9 +20,6 @@
 ## Getters / Views
 
 ### `.currentRoundIndex() -> uint`
-### `.getRound(uint index) -> Round`
```

Comment:
We are actively using this method as part of spark-api startup, see here:

https://github.com/filecoin-station/spark-api/blob/9fba4df7920c20abc8f740012a546bc346e245ac/lib/round-tracker.js#L35

Before you continue working on this PR, can you please first rework spark-api initialization to NOT depend on `getRound` method? We need to validate that it's feasible to get the current round by scanning the events emitted in the past.

Reply:
We're not using this method, we're using `currentRoundIndex()`, which is unchanged

Reply:
Ah, I see. Perfect 👍🏻